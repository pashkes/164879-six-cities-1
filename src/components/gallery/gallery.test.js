import React from "react";
import {shallow, configure} from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import Gallery from "./gallery.jsx";

configure({adapter: new Adapter()});

it(`gallery is renders correctly`, () => {
  const gallery = shallow(<Gallery photos={[``]} />);
  expect(gallery).toMatchSnapshot();
});
