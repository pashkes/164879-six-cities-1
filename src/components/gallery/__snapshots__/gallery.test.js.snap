// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gallery is renders correctly 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  >
    <div
      className="property__image-wrapper"
      key="0"
    >
      <img
        alt="studio"
        className="property__image"
        src=""
      />
    </div>
  </div>
</div>
`;
